deploy/helm/eunomia-operator/templates/deployment.yaml

```diff
@@ -1,5 +1,5 @@
 {{- with .Values.eunomia.operator }}
-{{- if .deployment.enabled -}}
+{{- if and (.deployment.enabled) (not .deployment.nsRbacOnly) -}}
```

Comment:
When initially I was using bare `go test` instead of `operator-sdk test local` I tried to use the `-test.parallel=4` flag in one run to see what happens but there was no difference in test time - we need `t.Parallel()`.

To enable parallelism in `operator-sdk test local` we can use the `--go-test-flags` flag to pass the option. The command by itself sets the parallel flag in [only one case](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/cmd/operator-sdk/test/local.go#L205-L207) (not ours, fortunately).

In the end `operator-sdk test local` is just [a wrapper for `go test`](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/cmd/operator-sdk/test/local.go#L211-L222) - [it just uses exec](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/internal/util/projutil/exec.go#L76-L86).